78 bug expression page design improvement

SujinCho7 opened the pull request:
Overall design, layout brush up.
Add image to desktop.

SujinCho7 commented:
Realize that the starting page of Button sizes are slightly different. keep this PR open. I'll come back here.

ebisuG commented:
@SujinCho7 
The conditional to post words learned is when user click next button, so the last content doesn't have an opportunity to send the request, because the button is hidden when user reaches the last content.

SujinCho7 commented:
@ebisuG this ticket is only for the style so I think we need to merge this and open a new issue about your comment.

SujinCho7 commented:
@ebisuG let's roll back to 5ce8cb3 at this moment and revisit the toast after merging.

ebisuG committed:
Revert "fix:Change the timing that toast is shown."

This reverts commit d3ed840ee10f464dac61a8891f572a2e188e996a.

ebisuG committed:
Revert "feat:Add react toastify"

This reverts commit 2b6bd33470dbba2e106edf8040f78ada3fa53cf4.

ebisuG committed:
merge with dev

ebisuG committed:
merge with dev

ebisuG committed:
merge with dev

the latest one that includes the modification in this branch has been already merged before this branch
